app/routes/joblistings/components/JoblistingEditor.css

```diff
@@ -14,10 +14,6 @@
   font-size: 24px;
 }
```

Comment:
If I haven't missed any usages this class could probably be deleted

Reply:
Nice catch! `.root` was not being used as well 🙃